lib/generate.js

```diff
@@ -135,6 +135,33 @@ generate.javascript.builtinPostprocessors = {
     "id": "id"
 }
 
+
+generate.es2015 = generate.es6 = function (parser, exportName) {
+    return "// Generated automatically by nearley\n" +
+        "// http://github.com/Hardmath123/nearley\n" +
+        "{\n" +
+        "const id = x => x[0];\n" +
+        parser.body.join('\n') +
+        "const grammar = {\n" +
+        "    Lexer: " + parser.config.lexer + ",\n" +
+        "    ParserRules: " + serializeRules(parser.rules, generate.es2015.builtinPostprocessors) + ",\n" +
+        "    ParserStart: " + JSON.stringify(parser.start) + "\n" +
+        "}\n" +
```

Comment:
I'm just a fellow contributor, but sure! There're no other PRs to the codebase at the moment, so no potential conflicts.

It may be a good idea to make an issue first and discuss code styleguide properly, because there's none and the style is all over the spectrum. If you'd champion that issue, I'd be a happy man. I currently only have time to slowly inch forward with the docs over at #254.